Source/Resources/Content/MSGraph/Groups/BUILTIN - Autopilot Devices - Self Deploying.json

```diff
@@ -1,6 +1,6 @@
 {
   "assignedLicenses": [],
-  "description": "Contains corporate devices tagged with 'Autopilot-SelfDeploying'. The tag is added by uploading the hardware hash including the column OrderId or by adding a Group Tag in the devices page of the Azure console. Not yet enrolled devices eligible for autopilot are identified as those having a 'ZTDId' marker.",
+  "description": "What is it: A dynamic device group that is used for assigning the Self-Deploying Intune Autopilot profile to devices. \n Simeon Notes: This group contains corporate physical devices with the tag: 'Autopilot-SelfDeploying'. Devices can be tagged from the Intune portal or via an automated hardware hash upload. To tag in Intune: Devices > Windows Enrollment > Devices, select the required device and add to the 'Group Tag' field. To tag via automated hardware hash included with column OrderId. \n Learn More: https://docs.microsoft.com/en-us/mem/autopilot/self-deploying",
```

Comment:
**What does this do?**
Creates a dynamic device group that is used for assigning self-deploying Intune Autopilot profiles to devices. This group contains corporate physical devices with the tag: 'Autopilot-SelfDeploying'. Devices can be tagged from the Intune portal or via an automated hardware hash upload. To tag in Intune go to Devices > Windows Enrollment > Devices, then select the required device and add to the 'Group Tag' field to tag via automated hardware hash included with column OrderId. 

**Why should you use this?**
This is used for assigning the Intune Autopilot self-deploying profile to devices. 

**What is the end user impact?**
Users will be able to be self-deployed when assigned this profile. 

**Learn more**
[Windows Autopilot Self-Deploying mode](https://docs.microsoft.com/en-us/mem/autopilot/self-deploying)